tests/test_clean.py

```diff
@@ -427,16 +427,18 @@ def test_clean_idempotent():
 
 def test_only_text_is_cleaned():
     some_text = 'text'
-    some_type = int
+    some_type = 42
```

Comment:
You've changed the value here so this should be `some_int`. I think we really want something of type type, too.

Reply:
The type of 42 is ```int```.
The type of ```int``` is ```type```. I can do specific cases for such type but I'm not sure it's really useful.

Do you prefer:
- I change the test to keep ```int``` instead of 42 in the test and update the assertion
- I add if/else for native types (```int```, ```float```, ```list```, etc.)

Reply:
You changed a variable value to something that doesn't match the variable name. Further, it's virtually the same thing as `some_text = 'text'`, so I don't think the change you made is very interesting.

Please change it back to `some_type = int` and then fix the test accordingly.